packages/ui/src/features/data-points/data-points-reducer.ts

```diff
@@ -156,9 +166,10 @@ const buildRows = (
       return {
         isNew: false,
         dataPoints: vars.concat(scores),
+        disabled: !item.meta.enabled,
         // Uncomment the following line to display a meta data property in the table
         // .concat([{ name: 'id', value: `${item.meta.id}` }]),
-      }
+      } as TableDataRow
```

Comment:
We should avoid type cast. The Correct way to ensure typing here is to modify line 149 like this `.map((item):TableDataRow => {`